sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 3.1.0-beta.3 (Unreleased)
+## 3.1.0-beta.3 (2021-04-06)
```

Comment:
not sure if u want to wait until that day to change this.
In .NET we always wait for the day of, but not sure Java

Reply:
It should be fine for Java.